packages/plugin/src/theia.d.ts

```diff
@@ -3703,10 +3949,10 @@ declare module '@theia/plugin' {
         [key: string]: boolean | number | string;
     }
 
-	/**
-     * The document formatting provider interface defines the contract between extensions and
-     * the formatting-feature.
-     */
+    /**
+        * The document formatting provider interface defines the contract between extensions and
```

Comment:
formatting issue here and at the function below

Reply:
fixed